### DISCUSSION TITLE
[Warning] Do not make Floorp issues about Windows that are no longer supported!

### DISCUSSION THREAD

**Discussion by creeper-0910** [Floorp-Projects/Floorp#943] [Announcements]
We understand that many users want to install Floorp on older Windows.
However, old Windows is already unsupported, even by Firefox, and forcing installation will cause many problems.
We have neither the time nor the funds to support or fix it.

Please do not ask developers to make Floorp installable on older Windows.
Do not make Floorp issues about Windows that are no longer supported!

Thank you for your understanding and cooperation.

**Comment by creeper-0910** [comment]
Please do not send Floorp issues related to older Windows to developer's email.
For developers it is an annoyance.

Thank you for your understanding and cooperation.

**Comment by EggToaster** [comment]
I confirmed that you still can use floorp by installing [VxKex](https://github.com/vxiiduu/VxKex).
This is unofficial method, and it does not support windows 8, and it might stop working after update.
